src/calendar/header/index.js

```diff
@@ -42,11 +42,17 @@ class CalendarHeader extends Component {
     if (!this.props.hideArrows) {
       leftArrow = (
         <TouchableOpacity onPress={this.substractMonth} style={this.style.arrow}>
-          <Image source={require('../img/previous.png')} style={this.style.arrowImage}/></TouchableOpacity>
+          {this.props.renderArrow
+            ? React.cloneElement(this.props.renderArrow('left'))
```

Comment:
actually that is a good point, in the last commit it was explicit `leftArrow` and `rightArrow` props so doing `React.cloneElement(this.props.leftArrow)` seemed better than `<this.props.leftArrow />`. Now that its a method there is no reason for it. Will remove. :smile: